SpatialGDK/Source/SpatialGDK/Public/Utils/EntityFactory.h

```diff
@@ -5,12 +5,10 @@
 #include "SpatialCommonTypes.h"
 #include "Utils/SpatialStatics.h"
 
-#include <WorkerSDK/improbable/c_schema.h>
```

Comment:
Why are these includes removed?

Reply:
Worker types are wrapped in Unreal types, and included via SpatialCommonTypes